modules/programs/navi.nix

```
@@ -0,0 +1,97 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+
+  cfg = config.programs.navi;
+
+  yamlFormat = pkgs.formats.yaml { };
+
+  inherit (pkgs.stdenv.hostPlatform) isDarwin;
+
+in {
+  meta.maintainers = [ maintainers.marsam ];
+
+  options.programs.navi = {
+    enable = mkEnableOption "Navi";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.navi;
+      defaultText = literalExpression "pkgs.navi";
+      description = "The package to use for the navi binary.";
+    };
+
+    settings = mkOption {
+      type = yamlFormat.type;
+      default = { };
+      defaultText = literalExpression "{ }";
+      example = literalExpression ''
+        {
+          cheats = {
+            paths = [
+              "~/cheats/"
+            ];
+          };
+        }
+      '';
+      description = ''
+        Configuration written to
+        <filename>~/.config/navi/config.yaml</filename> on Linux
+        or <filename>~/Library/Application Support/navi/config.yaml</filename> on Darwin. See
+        <link xlink:href="https://github.com/denisidoro/navi/blob/master/docs/config_file.md"/>
+        for more information.
+      '';
+    };
+
+    enableBashIntegration = mkOption {
+      default = true;
+      type = types.bool;
+      description = ''
+        Whether to enable Bash integration.
+      '';
+    };
```

I think this is cleaner (also for the other `enable` flags`):

```suggestion
    enableBashIntegration = mkEnableOption "Bash integration" // { default = true; };
```

---

Fixed. Thanks!